src/test/java/htsjdk/tribble/TestUtils.java

```diff
@@ -18,11 +18,30 @@
 
 package htsjdk.tribble;
 
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 /**
  * User: jacob
  * Date: 2012-Dec-13
  */
 public class TestUtils {
     public static String DATA_DIR = "src/test/resources/htsjdk/tribble/";
+
+
+    public static Path getTribbleFileInJimfs(String vcf, String index, FileSystem fileSystem) throws IOException, URISyntaxException {
```

Comment:
Add a javadoc explaining that the `FileSystem` should be a `JimfsFileSystem`. It took me a while to figure out what was it.

Another option is to provide a destination `JimfsPath` instead to get the `FileSystem` from it.